### DISCUSSION TITLE
Unable to see login screen after logoff when installed as a service

### DISCUSSION THREAD

**Discussion by De4dm4sterBR** [rustdesk/rustdesk#8649] [General]
Hi,

Currently i don't know if its supposed to work this way, but unlike other remote access softwares (like AnyDesk), when the application is installed as a service and the user logs off, we cant see the Windows Login Screen to manually change users or login again with the same user.

If is the way it works right now, i would like it to be a feature request and be able to see Windows Login Screen, because the way it works right now limits a lot our process of supporting our customers (not only the need for the user to login manually, but if we need to change to another user, like the local machine user or a user with elevated rights, it needs interaction on the other end, plus the fact the password change every time a connection is closed).

Thanks in advance!

**Comment by rustdesk** [comment]
It is supposed to see the login screen since the first version of RustDesk. Usually when you log off, the connection will be closed and it will reconnect automatically and then show the login screen (this takes a little while).

@21pages  follow it up.

**Comment by 21pages** [comment]
Could you try different versions?  https://github.com/rustdesk/rustdesk/releases, we wanna confirm which version cause you problem first.

**Comment by De4dm4sterBR** [comment]
> Could you try different versions? https://github.com/rustdesk/rustdesk/releases, we wanna confirm which version cause you problem first.

We are currently using a Custom Build ver 1.2.5.

When the user logs off, it briefly disconnects and asks for the password again (wich has changed since last connection), so we cant connect again.

**Comment by De4dm4sterBR** [comment]
And my issue is related to rustdesk-pro, since its a custom client. This move is correct?

**Comment by rustdesk** [comment]
> asks for the password again
What password are you using? permanent password or one-time password?

**Comment by De4dm4sterBR** [comment]
> > asks for the password again
> > What password are you using? permanent password or one-time password?

One time password, because permanent password gives that warning message and because of that its not suitable for us to use on our customers.

**Comment by rustdesk** [comment]
It is expected for one time password in log-off case.

**Reply by De4dm4sterBR** [reply]
> It is expected for one time password in log-off case.


Right, but there isn't a way for it to detect a session previously disconnected because of logoff and login again without asking for password or acceptance (like AnyDesk)? 

Or, there's a way to disable that warning on permanent password? it would solve our problems, but we cant open the software in our customers with that big warning on the screen (we tried and people actually freaks out thinking its a security issue).

**Reply by rustdesk** [reply]
The warning will be solved by this Friday.

**Reply by De4dm4sterBR** [reply]
Because we are actually without options here, one time password isn't a problem for us, but is impossible to reconnect again after logoff with this, and permanent password is unusable because of that warning.

**Reply by De4dm4sterBR** [reply]
> The warning will be solved by this Friday.

Oh, thanks! It will solve our problems.

**Comment by rustdesk** [comment]
Close and lock to favor https://github.com/rustdesk/rustdesk-server-pro/discussions/286#discussioncomment-9987493, we will update there.
